core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQLActionBuilder.kt

```diff
@@ -79,27 +91,127 @@ object GraphQLActionBuilder {
 
```

Comment:
would the code above deal with recursion? eg, an object with objects inside, like arrays of objects, and so on